shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/remote/ApiClient.kt

```diff
@@ -35,27 +41,20 @@ class ApiClient(
             }
 
             install(ContentNegotiation) {
-                json(
-                    Json {
-                        prettyPrint = true
-                        isLenient = true
-                        encodeDefaults = true
-                        ignoreUnknownKeys = true
-                    }
-                )
+                json(json)
             }
         }
     }
 
-    inline fun <reified T> body(builder: HttpRequestBuilder): Flow<T> {
-        return flow {
-            val data = httpClient.request(
-                builder.apply {
-                    contentType(ContentType.Application.Json)
-                }
-            ).bodyAsText()
-            // TODO Parse JSON:API here
-            emit(data as T)
-        }
+    suspend inline fun <reified T> post(path: String, requestBody: Any): T {
```

Comment:
So you will need another fun `get(path: String, requestParameters: Any)`? Also for other methods (PATCH, PUT,...)?

Reply:
@suho exactly 👍 I might need to move the shared stuff to a new `request` fun.